rust/Makefile

```diff
@@ -14,9 +18,13 @@ quiet_cmd_rustdoc = RUSTDOC $<
 	$(RUSTDOC) $(filter-out --emit=%, $(rustc_flags)) \
 		$(rustdoc_target_flags) -L $(objtree)/rust/ \
 		--output $(objtree)/rust/doc --crate-name $(subst rustdoc-,,$@) \
-		-Fmissing-docs @$(objtree)/include/generated/rustc_cfg $<
+		@$(objtree)/include/generated/rustc_cfg $<
 
-rustdoc: rustdoc-module rustdoc-compiler_builtins rustdoc-kernel
+rustdoc: rustdoc-core rustdoc-module rustdoc-compiler_builtins rustdoc-alloc rustdoc-kernel
+
+rustdoc-core: private skip_clippy = 1
```

Comment:
`skip_clippy` is not used for `rustdoc`. Did you add it for a particular reason?

Reply:
Sorry I didnt aware that. Fixed it!

Reply:
No need to say sorry! Thanks for working on this! :-)